docs/api/reading.md

```diff
@@ -287,7 +287,8 @@ Would embed in the `client` key the row referenced with `client_id`.
 
 <div class="admonition note">
     <p class="admonition-title">Design Consideration</p>
-    <p>In order for this feature to work as expected after a schema change, PostgREST currently requires to be restarted.</p>
+    <p>In order for this feature to work as expected after a schema change, PostgREST currently requires a restart.</p>
+    <p>Additionally, in versions >=0.3.1.1 views containing sub-select statements as columns will not have foreign key relations matched. Sub-select statements may occur in a WHERE clause following a JOIN condition. In previous versions, foreign key relations in views with columns defined with sub-selects were sometimes incidentally recognized.  </p>
```

Comment:
Let's not introduce confusion by mentioning versions and my saying something like "it used to work", it didn't :). That was a bug, while sometimes it made a correct match (by pure chance), most of the time, the relations were badly detected (between tables that did not have any relation).

Let's just say something like "Relation detection (based on foreign keys) are always reliably detected for tables. Views have a limitation, relation detection only works then there are no sub-selects in SELECT/FROM/JOIN part of the query, they are allowed in WHERE clause"